lib/form.js

```diff
@@ -204,39 +206,59 @@ Form.prototype.track_cta = function (elements, events, type) {
   this.track_element(elements, events, 'request', handler)
 }
 
-// Form.prototype.track_submit = function(forms, event, properties) {
-//   if (!forms) return this;
-//   // always arrays, handles jquery
-//   if (type(forms) === 'element') forms = [forms];
-//
-//   each(el => {
-//     if (type(el) !== 'element') throw new TypeError('Must pass HTMLElement to `track_submit`.');
-//     function handler(e) {
-//       prevent(e);
-//
-//       var ev = is.fn(event) ? event(el) : event;
-//       var props = is.fn(properties) ? properties(el) : properties;
-//
-//       // self.track(ev, props);
-//       console.log(ev, props);
-//
-//       this._callback(function() {
-//         el.submit();
-//       });
-//     }
-//
-//     // Support the events happening through jQuery or Zepto instead of through
-//     // the normal DOM API, because `el.submit` doesn't bubble up events...
-//     var $ = window.jQuery || window.Zepto;
-//     if ($) {
-//       $(el).submit(handler);
-//     } else {
-//       on(el, 'submit', handler);
-//     }
-//   }, forms);
-//
-//   return this;
-// };
+Form.prototype.load_external_script = function (src) {
```

Comment:
Convenience function to load an external script (with no assistance from jQuery)